src/resolver.rs

```diff
@@ -1203,18 +1242,19 @@ impl<'a> AuditGraph<'a> {
                     let to_ver = Some(&publisher.version);
                     let criteria = criteria_mapper.criteria_from_list(&entry.criteria);
                     let origin = DeltaEdgeOrigin::Trusted { publisher_index };
+                    let freshness = DeltaEdgeFreshness::fresh_publisher(publisher.is_fresh_import);
```

Comment:
I'm not sure we want to use `fresh_publisher` here. Technically it is somewhat "correct", but a trusted entry is less validated than a normal audit, so we might not want to be prioritizing them in this way?

I believe that if we put them at the same caveat level as other audits, it'll be picked below normal audits due to the tie-breaker ordering by `DeltaEdgeOrigin`, which is what we want:

https://github.com/mozilla/cargo-vet/blob/67e92b6d03f48802907b2c80119ed328289a114d/src/resolver.rs#L301-L302

https://github.com/mozilla/cargo-vet/blob/67e92b6d03f48802907b2c80119ed328289a114d/src/resolver.rs#L1486

```suggestion
                    let freshness = DeltaEdgeFreshness::fresh(publisher.is_fresh_import);
```